pkgs/development/python-modules/vfblib/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  fetchFromGitHub,
+  gitUpdater,
+  buildPythonPackage,
+  setuptools,
+  fonttools,
+  typing-extensions,
+  ufonormalizer,
+  ufolib2,
+  defcon,
+}:
+
+buildPythonPackage rec {
+  pname = "vfblib";
+  version = "0.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "LucasFonts";
+    repo = "vfbLib";
+    rev = "v${version}";
+    hash = "sha256-1F7Em3qX5QKHaGDYVfEOHwYu3PxZUWboe67Hgitvebc=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    fonttools
+    typing-extensions
+    ufonormalizer
+    ufolib2
+    defcon
+  ];
+
```

Comment:
Could you run tests with `pytestCheckHook` and `pythonImportsCheck`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
Sure